adapters/ix/ix.go

```diff
@@ -19,7 +19,8 @@ import (
 )
 
 type IxAdapter struct {
-	URI string
+	URI        string
+	bidderName string
```

Comment:
@ccorbo  could you help to understand how adding `bidderName` facilates privacy sandbox support

Reply:
Removed bidderName - not needed

Reply:
> Removed bidderName - not needed

@ccorbo  could you link commit where `bidderName` is removed. PR diff still has `bidderName`

Reply:
whoops sorry - updated now